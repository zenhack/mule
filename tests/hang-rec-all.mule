#SKIP
# FAILING: This curently hangs the type checker.
4 : rec a. all b. a
