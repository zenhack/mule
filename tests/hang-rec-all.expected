
Desugared:

	((: (rec a (all b a))) (Integer 4))
Type error: mismatched guards
